libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionNoSetupTest.kt

```diff
@@ -263,6 +265,61 @@ class MapboxTripSessionNoSetupTest {
             assertEquals(listOf("1", "1"), newVoiceInstructions)
         }
 
+    @Test
+    fun `banner instruction is transferred from NN to SDK status update`() =
+        coroutineRule.runBlockingTest {
+            // arrange
```

Comment:
Minor thing: What about removing the `// arrange` `// act` `// assert` comments and just mark the Triple A with new lines between the 3 blocks?

Reply:
The test has an empty line in act, maybe that's a sign that the act sections is too big 🤔 
I agree with you in general but for this specific test I think the comments work better 🙂